contributing.rst

```diff
@@ -22,7 +22,7 @@ Open pull requests
 ==================
 
 Look through the currently open pull requests in the collection repository.
-You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_.
+You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_. Also if you want to test a pull request locally, refer to the `test PR guide <test_pr_locally_guide.rst>`.
```

Comment:
```suggestion
You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_. Also if you want to test a pull request locally, refer to the `test PR locally guide <test_pr_locally_guide.rst>`.
```

Reply:
I asked Amin to remove "locally" (too many "locally" IMO) but this variant is also fine:)

Reply:
Well, it's the title of that document ;-) Maybe we should remove the `locally` before the comma in this sentence, so we can use the correct title here?

Reply:
Sounds good to me:)

Reply:
@felixfontein could you please adjust the proposed changes?

Reply:
@Andersson007 done